packages/astro/src/content/utils.ts

```diff
@@ -228,7 +228,7 @@ export async function loadContentConfig({
 	}
 	try {
 		unparsedConfig = await tempConfigServer.ssrLoadModule(
-			decodeURIComponent(contentPaths.config.pathname)
+			fileURLToPath(contentPaths.config.pathname)
```

Comment:
Meant to do 
```js
new URL(fileURLToPath(contentPaths.config)).pathname
```
But that results in the majority of tests to fail. I'll revert back to using `decodeURIComponent`

Reply:
Are tests failing with just `fileURLToPath(contentPaths.config.pathname)`? (without the URL wrapper)?

Reply:
Yes, in both cases

Reply:
Oh sorry, I would expect it to be `fileURLToPath(contentPaths.config)`. config is a file URL. That converts it to a regular path which Vite can take.

Reply:
Essentially you almost never need/want to use `.pathname` anywhere that takes a file path.

Reply:
Ah thanks for your help, that did it!